browse-scaler/src/index.js

```diff
@@ -1,66 +1,112 @@
-const { slurpImageIntoBuffer, resizeImage, notFound } = require('./resize');
-const { getCollectionLevelBrowseImage, getGranuleLevelBrowseImage } = require('./cmr');
-const { cacheImage, getImageFromCache } = require('./cache');
+const { slurpImageIntoBuffer, resizeImage, notFound } = require("./resize");
+const {
+  getCollectionLevelBrowseImage,
+  getGranuleLevelBrowseImage
+} = require("./cmr");
+const { cacheImage, getImageFromCache } = require("./cache");
 
 const buildResponse = (image, format) => {
+  console.log(`Image for response: ${image}`);
+  console.log(`Image format: ${format}`);
   return {
     statusCode: 200,
     headers: {
-      'Content-Type': format
+      "Content-Type": format
     },
     body: image,
     isBase64Encoded: true
   };
 };
 
-const getImageUrlFromConcept = async conceptId => {
-  if (conceptId.startsWith('G')) {
+const getImageUrlFromConcept = async (conceptId, conceptType) => {
+  console.log(`Concept id: ${conceptId}`);
+  if (conceptId === null || conceptId === undefined) {
+    return null;
+  }
+
+  if (conceptType === "granules") {
     return getGranuleLevelBrowseImage(conceptId);
   }
 
-  if (conceptId.startsWith('C')) {
+  if (conceptType === "datasets") {
     return getCollectionLevelBrowseImage(conceptId);
   }
 
   console.log(`Unable to find browse imagery for concept: ${conceptId}`);
   return null;
 };
 
-const resizeImageFromConceptId = async (conceptId, height, width) => {
+const resizeImageFromConceptId = async (
+  conceptType,
+  conceptId,
+  height,
+  width
+) => {
   // If given an image url, fetch the image and resize. If no image
   // exists, return the not found response
-  const imageUrl = await getImageUrlFromConcept;
+  const imageUrl = await getImageUrlFromConcept(conceptId, conceptType);
 
   if (imageUrl === null) {
-    const cachedSvg = await getImageFromCache('NOT-FOUND');
-    if (cachedSvg) {
-      return buildResponse(cachedSvg, 'image/svg');
-    }
+    // const cachedSvg = await getImageFromCache("NOT-FOUND");
```

Comment:
Document why this is commented out. Write a ticket and add a ticket number here in the code if it needs to be fixed. I'm assuming you are disabling caching here, and that you'll enable it with another ticket.